crates/nargo_cli/tests/test_data_ssa_refactor/brillig_unitialised_arrays/src/main.nr

```diff
@@ -0,0 +1,13 @@
+
+fn main(x: Field, y: Field) -> pub Field  {
```

Comment:
This test is in `test_data_ssa_refactor` which is outdated, can we move this into `execution_success`?